tsl/src/compression/create.c

```diff
@@ -1057,28 +1087,31 @@ tsl_process_compress_table(AlterTableCmd *cmd, Hypertable *ht,
  * This function specifically adds the column to the internal compression table.
  */
 void
-tsl_process_compress_table_add_column(Hypertable *ht, ColumnDef *orig_def)
+tsl_process_compress_table_add_column(Hypertable *ht, const AlterTableCmd *cmd)
```

Comment:
Why do you want to tie this to an ALTER TABLE command? We should be able to add a column on demand if we like. I don't see what `missing_ok` offers us here. In any case, if we really needed `missing_ok` we could add an extra `bool` argument.

In this case if `missing_ok` is true, then it is a no-op that prints a warning, otherwise it's an exception. Is this the idea here?